src/core/syntax-tree/syntaxer.ts

```diff
@@ -64,6 +64,10 @@ export default (lexerResult: ILexerResult): ISyntaxerResult => {
         return { valid: false, error: `lexer -- ${lexerResult.error}` };
     }
 
+    if (lexerResult.lexemes.length === 0) {
+        return { valid: true };
+    }
```

Comment:
Having no query string is now a valid / null tree case instead of an invalid case